packages/conform-valibot/tests/coercion/schema/array.test.ts

```diff
@@ -24,14 +24,14 @@ describe('array', () => {
 		});
 
 		const schema2 = object({ nest: array(object({ name: string() })) });
-		const formData2 = createFormData('nest[].name', 'test name');
+		const formData2 = createFormData('nest[0].name', 'test name');
 		const output2 = parseWithValibot(formData2, { schema: schema2 });
 		expect(output2).toMatchObject({
 			status: 'success',
 			value: { nest: [{ name: 'test name' }] },
 		});
 
-		const errorFormData = createFormData('nest[].name', '');
+		const errorFormData = createFormData('nest[0].name', '');
```

Comment:
@chimame FYI: you may need to update the tests in your repo. We originally added empty-bracket support to make it easier to resolve validation attributes from the derived constraints, but empty brackets aren't officially supported when parsing formdata as it is ambiguous which item you are referring to in a nested object.

With this refactor, we have added limited empty-bracket support: if we see `items[]` at the **end** of a path, we will initialize an array and append the value. But empty brackets anywhere else in the path remain unsupported.

Reply:
Thank you for letting me know. 👍 
I'll keep that in mind when I create tests in the future. I'll also try to change any existing tests that seem to need changing.